Cargo.toml

```
@@ -4,7 +4,7 @@ name = "http"
 # - Update html_root_url in lib.rs.
 # - Update CHANGELOG.md.
 # - Create git tag
-version = "0.1.13"
+version = "0.1.14"
```

```suggestion
version = "0.1.13"
```

Mind leaving the version change out? We can bump it as part of the changelog generation commit.

---

Oh sure :)